addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -254,17 +254,26 @@
                             <tr t-att-class="'bg-200 font-weight-bold o_line_section' if line.display_type == 'line_section' else 'font-italic o_line_note' if line.display_type == 'line_note' else ''">
                                 <t t-if="not line.display_type" name="account_invoice_line_accountable">
                                     <td name="account_invoice_line_name">
-                                        <t t-set="translation_name" t-value="line.with_context(lang='ar_001').product_id.name"/>
                                         <t t-if="line.product_id">
-                                            <span t-field="line.product_id.name" t-options="{'widget': 'text'}"/>
-                                            <t t-if="line.product_id.name != translation_name">
+                                            <t t-set="arabic_name" t-value="line.with_context(lang='ar_001').product_id.name"/>
+                                            <t t-set="english_name" t-value="line.with_context(lang='en_US').product_id.name"/>
+
+                                            <span t-field="line.with_context(lang='en_US').product_id.display_name"
+                                                t-options="{'widget': 'text'}"/>
+
+                                            <t t-if="arabic_name != english_name">
                                                 <br/>
                                                 <span t-field="line.with_context(lang='ar_001').product_id.name"
-                                                      t-options="{'widget': 'text'}"/>
+                                                    style="text:right"
+                                                    t-options="{'widget': 'text'}"/>
+                                            </t>
+
+                                            <t t-if="line.name != line.product_id.display_name">
```

Comment:
```suggestion
                                            <t t-if="line.name != line.product_id.display_name and line.name != line.with_context(lang='ar_001').product_id.display_name">
```
I would think that the Arabic name would still be double without this...
And why not use display_name for arabic_name already?
@HANNICHE-Walid

Reply:
![image](https://user-images.githubusercontent.com/46480423/207014114-c5cfcaa2-0f18-4369-9812-ba567621ef39.png)